features/networking/multus.feature

```diff
@@ -417,3 +417,34 @@ Feature: Multus-CNI related scenarios
     And the output should contain "net2"
     And the output should contain 2 times:
       | macvlan mode bridge |
+
+
+  # @author weliang@redhat.com
+  # @case_id OCP-25676
+  @admin
+  Scenario: Supported runtimeConfig/capability for MAC/IP
+    # Make sure that the multus is enabled
+    Given the multus is enabled on the cluster
+
+    # Create the net-attach-def via cluster admin
+    Given I have a project
+    When I run the :create admin command with:
+      | f | https://raw.githubusercontent.com/weliang1/Openshift_Networking/master/Features/multus/runtimeconfig-def-ipandmac.yaml |
+      | n | <%= cb.project_name %>                                                                                                 |
+    Then the step should succeed
+
+    #Clean-up required to erase above net-attach-def after testing done
+    Given I register clean-up steps:
```

Comment:
437-444 can be replaced as
`Given admin ensures "runtimeconfig-def" network_attachment_definition is deleted from the "<%= cb.project_name %>" project after scenario`

but post #427 merge though

Reply:
Can we get #427 merged first? Then I can update this PR and continue other test cases requesting this procedure too, Thanks!

Reply:
@weliang1 #427 is merged now.

Reply:
@anuragthehatter Updated it to use new way from your lib, thanks!